### PR TITLE
Adding app installation link

### PR DESCRIPTION
**Pull request by shigupt202**
Adding link for installing the app in check run and action logs.

image

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -25,6 +25,8 @@ const trivyHelper = __importStar(require("./trivyHelper"));
 const fileHelper = __importStar(require("./fileHelper"));
 const githubClient_1 = require("./githubClient");
 const httpClient_1 = require("./httpClient");
+const APP_NAME = 'Scanitizer';
+const APP_LINK = 'https://github.com/apps/scanitizer';
```
Comment (ds-ms) [inline lib/utils.js:29]: This is not published yet?

Comment (shigupt202) [reply lib/utils.js:29]: The app is not public yet.

File: src/utils.ts [hunk 2]
```diff
@@ -149,6 +154,9 @@ function getCheckSummary(trivyStatus: number, dockleStatus: number): string {
     summary = `${summary}\n\n${dockleSummary}`;
   }
 
+  let appHyperlink = `<a href=${APP_LINK}>${APP_NAME}</a>`;
+  summary = `${summary}\n\nFor a better experience of handling the detected vulnerabilities and best practice violations, install ${appHyperlink}.`
```
Comment (ajinkya599) [inline src/utils.ts:158]: Just a thought:
"For a better experience in managing allowedlist, install scanitizer app"

File: src/utils.ts [hunk 3]
```diff
@@ -19,6 +22,8 @@ export async function createScanResult(trivyStatus: number, dockleStatus: number
     && response.body.message.error_code === 'APP_NOT_INSTALLED') {
     // If the app is not installed, try to create the check run using GitHub actions token.
     console.log('Looks like the scanitizer app is not installed on the repo. Falling back to check run creation through GitHub actions app...');
+    console.log(`For a better experience of handling the detected vulnerabilities and best practice violations, install ${APP_NAME} from ${APP_LINK}.`);
```
[the remark this answers is not in the document]
Comment (shigupt202) [reply src/utils.ts:25]: I think you mean `${APP_NAME} app from ${APP_LINK}`?

**Pull request merged**